packages/eslint-plugin-lit-a11y/tests/lib/rules/autocomplete-valid.js

```diff
@@ -10,18 +10,19 @@
 
 const { RuleTester } = require('eslint');
 const rule = require('../../../lib/rules/autocomplete-valid');
-const { prependLitHtmlImport } = require('../../../lib/utils/utils.js');
 
 //------------------------------------------------------------------------------
 // Tests
 //------------------------------------------------------------------------------
 
 const ruleTester = new RuleTester({
+  settings: { litHtmlSources: false },
```

Comment:
why do we have false as the default setting everywhere in the tests?

Reply:
To disable checking, so as not to pollute tests

Reply:
in other words, this takes the place of `valid: [...].map(prependLitHtmlImport)`